### DISCUSSION TITLE
Could it be useful to add in the Ground-Truth Preview window the possibility to set the PSM?

### DISCUSSION THREAD

**Discussion by Blu3train** [buliasz/tesstrain-windows-gui#23] [Ideas]
Even if I do the training with the PSM set to 13, the program with which I use the generated model has a different PSM, specifically it is set to 6.
What do you think if you add a box to be able to set the PSM in the Ground-Truth Preview window?
So we can compare the difference in output by changing the value.

**Comment by buliasz** [comment]
Done in the new 5.9 version.

**Comment by Blu3train** [comment]
Thanks
